### PR TITLE
[Merged by Bors] - Expand documentation

### PR DESCRIPTION
**Pull request by kebekus**
doc: Minor improvement in tensor product documentation

This PR adds names to the objects mentioned in the page intro so users can more easily navigate to the relevant definitions and theorems in the page body.

---
[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/LinearAlgebra/TensorProduct/Basic.lean [hunk 1]
```diff
@@ -13,22 +13,25 @@ import Mathlib.Tactic.SuppressCompilation
 /-!
 # Tensor product of modules over commutative semirings.
 
-This file constructs the tensor product of modules over commutative semirings. Given a semiring
-`R` and modules over it `M` and `N`, the standard construction of the tensor product is
-`TensorProduct R M N`. It is also a module over `R`.
+This file constructs the tensor product of modules over commutative semirings. Given a semiring `R`
+and modules over it `M` and `N`, the standard construction of the tensor product is `TensorProduct R
+M N`. It is also a module over `R`.
```
Comment (fpvandoorn) [inline Mathlib/LinearAlgebra/TensorProduct/Basic.lean:18]: please no line break inside a code block

**Review by fpvandoorn — APPROVED** [review]
LGTM

File: Mathlib/LinearAlgebra/TensorProduct/Basic.lean (hunk 1 above)
Comment (kebekus) [reply Mathlib/LinearAlgebra/TensorProduct/Basic.lean:18]: Roger. Is there anything I need to do now?

Comment (kebekus) [reply Mathlib/LinearAlgebra/TensorProduct/Basic.lean]: I fixed the line break issue, as requested.

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

**Pull request closed without merging**